service/dap/server.go

```diff
@@ -999,7 +1002,6 @@ func (s *Server) stopDebugSession(killProcess bool) error {
 		switch err.(type) {
 		case proc.ErrProcessExited:
 			exited = err
```

Comment:
We could also send ExitedEvent for all the case where we detect that process has Exited, but I have no idea how that would change/help. The event takes an exit code and I thought maybe vscode would reflect that somehow, but in my experiments I did not see any impact. I also checked a number of implementations form the dap spec page and did not find any usage there or in the mock debugger.

Reply:
There is no corresponding UI for exited events (at least in VS Code) at this point. If this is something important to Go users we should probably file an issue with them.

Reply:
We already print to the debug console when process exited, so I don't think it is very important to have special support in the UI.